server/src/main/java/school/journal/repository/impl/ClazzRepository.java

```diff
@@ -23,7 +23,8 @@ public Clazz create(Clazz clazz, Session session) throws RepositoryException {
 
     @Override
     public Clazz update(Clazz clazz, Session session) throws RepositoryException {
-        session.update(clazz);
+        session.merge(clazz);
```

Comment:
because we've used loading object. it causes exception, that means we have two object in session, but one we update, other became invalid and hibernate reports about it.